### PR TITLE
chore: Updated PostGresQL to version 15

### PR DESCRIPTION
**Pull request by abellotti**
- Updating the PostGresQL quadlet service to use version 15

### CODE REVIEW

File: config/discovery-db.container [hunk 1]
```diff
@@ -5,7 +5,7 @@ PartOf=discovery-app.service
 
 [Container]
 ContainerName=discovery-db
-Image=registry.redhat.io/rhel8/postgresql-12:latest
+Image=registry.redhat.io/rhel8/postgresql-15:latest
```
Comment (infinitewarp) [inline config/discovery-db.container:8]:
```suggestion
Image=registry.redhat.io/rhel9/postgresql-15:latest
```

Comment (abellotti) [reply config/discovery-db.container:8]: Ahh, should be good too 👍 . Let me try it.

Comment (infinitewarp) [reply config/discovery-db.container:8]: QE has been using `rhel9/postgresql-15` in their CI for a few weeks. I'm pretty sure it's stable. 🙂

Comment (abellotti) [reply config/discovery-db.container]: Just tested it with Quadlets @infinitewarp works fine. Just pushed an update.

**Review by bruno-fs — APPROVED** [review]

**Review by infinitewarp — APPROVED** [review]

**Pull request merged**